db/seeds.rb

```diff
@@ -5,3 +5,12 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+users = %w(macio basia seba mati brajan szefu lysy adik jan janusz)
+
+users.each_with_index do |user, index|
```

Comment:
you're not using the `index` variable here, we can remove it and change the method then

Reply:
corrected